app/src/main/java/com/locoquest/app/dao/IBenchmarkDAO.kt

```diff
@@ -14,29 +14,3 @@ interface IBenchmarkDAO {
     fun getBenchmarkByPid(@Query("pid") pid: String): Call<Benchmark>
 }
 
-fun main() {
```

Comment:
Moved RetrofitBuilder from DAO to the Service file.

Reply:
Yeah, I'd probably move it to service.